packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerPlugin.m

```diff
@@ -8,9 +8,9 @@
 #import <AVFoundation/AVFoundation.h>
 #import <GLKit/GLKit.h>
 
-#import "AVAssetTrackUtils.h"
-#import "FVPDisplayLink.h"
-#import "messages.g.h"
+#import "./include/video_player_avfoundation/AVAssetTrackUtils.h"
+#import "./include/video_player_avfoundation/FVPDisplayLink.h"
+#import "./include/video_player_avfoundation/messages.g.h"
```

Comment:
[Comment from previous PR](https://github.com/flutter/packages/pull/6634#discussion_r1589475560) by @stuartmorgan:
Do we know why include paths aren't taking care of this?

Reply:
So for most of the headers, include paths is sufficient. However, `messages.g.h` is a problem when there's multiple plugins that also have a `messages.g.h`. Seems like it can't resolve which to use. More explanation here: https://github.com/flutter/flutter/issues/147587#issuecomment-2086461349

So I decided to change all of them to relative paths to keep it uniform

Reply:
I'm a little surprised the compilation doesn't scope the search to just the module. Maybe it's the way we have to make all the headers public to make our unit tests work?

We could consider using a different naming convention for Obj-C out from Pigeon in our repo to give them unique names to avoid this, but relative paths are fine too.